docs/helpers.py

```diff
@@ -0,0 +1,106 @@
+import glob
+from pathlib import Path
+from typing import Any, Iterable, List, Optional
+
+
+class QuartoPrint(List[str]):
+    def __init__(self, data: Iterable[str]):
+        super().__init__(data)
+
+    def __str__(self) -> str:
+        return "\n".join(str(item) for item in self)
+
+    def append_file(self, file_path: str, file_name: Optional[str] = None):
+        if file_name is not None:
+            self.append(f"## file: {file_name}")
+
+        with open(file_path, "r") as app_file:
+            app_contents = app_file.read()
+            self.append(app_contents)
+
+
+def list_files(path: str) -> List[str]:
+    files = glob.glob(path + "/**", recursive=True)
+    files = [file for file in files if not glob.os.path.isdir(file)]
+    return files
+
+
+def include_shiny_folder(
+    path: str,
+    file_name: str = "app.py",
+    exclusions: List[str] = [],
+    components: str = "editor, viewer",
+    viewer_height: str = "800",
+    extra_object: Any = "",
+) -> None:
+    print(
+        _include_shiny_folder(
+            path, file_name, exclusions, components, viewer_height, extra_object
+        )
+    )
```

Comment:
Feels like this should get removed (and if you want to `print()` for local debugging, just `print(block)` at the end of the function)?

Reply:
Oh this is designed to include a single shinylive app in a block. So you might want to use an app in tabs, but maybe also refer to just one version of the file in a standalone app without tabs. I added some doc strings to clarify that.